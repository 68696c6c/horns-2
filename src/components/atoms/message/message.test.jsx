/* global describe, it, expect */
import React from 'react'

import { render } from 'setup-test'

import Message from '.'

describe('Message', () => {
  it('should render as default', () => {
    const { container } = render(<Message>example</Message>)
    expect(container).toMatchSnapshot()
  })
})
