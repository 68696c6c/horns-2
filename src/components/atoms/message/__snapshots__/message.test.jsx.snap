// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message should render as default 1`] = `
.emotion-0 {
  color: rgb(23,162,184);
  border-color: rgb(23,162,184);
  font-family: Helvetica,sans-serif;
  font-size: medium;
  font-style: normal;
  font-weight: 400;
  line-height: 1em;
}

<div>
  <label
    class="emotion-0"
    color="info"
  >
    example
  </label>
</div>
`;
